app/create_db_main.py

```diff
@@ -37,13 +37,13 @@
 
     db_dict = db_to_dict(db)
 
-    out = open('db_output.json', "a")
-    json_db = json.dumps(db_dict)
+    out = open('IrishCorpus/db_output.json', "a", encoding='utf-8')
```

Comment:
I don't think we want to add a directory to the output path. For one this can be used for any language not just Irish and also depending on where you call this function from, there may or may not be an existing corpus directory.

I would say we either just output the file to wherever the script is called from or add another argument option to specify where we want to output the file.

Reply:
In all honesty, I forgot that I added this, itll be removed